app/assess/models/fund_summary.py

```diff
@@ -69,6 +71,12 @@ def create_fund_summaries(fund: Fund) -> list[FundSummary]:
                     round_short_name=round.short_name.lower(),
                 ),
                 access_controller=AssessmentAccessController(fund.short_name),
+                export_href=url_for(
+                    "assess_bp.assessor_export",
+                    fund_short_name=fund.short_name,
+                    round_short_name=round.short_name.lower(),
```

Comment:
Any specific reason we go `.lower()` here?

Reply:
just left over code that needed removing, good catch